lib/WorkflowMain.groovy

```diff
@@ -21,7 +23,7 @@ class WorkflowMain {
     //
     // Generate help string
     //
-    public static String help(workflow, params, log) {
+    public static String help(workflow, params) {
         def command = "nextflow run ${workflow.manifest.name} --input samplesheet.csv --genome GRCh37 -profile docker"
```

Comment:
maybe make this GRCh38, just to keep with the times?

Reply:
Good idea thanks!